apis/r/src/rinterface.cpp

```diff
@@ -8,104 +8,113 @@
 #include <tiledb/tiledb_experimental>
 #endif
 
-#include "rutilities.h"         // local declarations
-#include "xptr-utils.h"         // xptr taggging utilities
+#include "rutilities.h"  // local declarations
+#include "xptr-utils.h"  // xptr taggging utilities
 
 // (Adapted) helper functions from nanoarrow
 //
 // Create an external pointer with the proper class and that will release any
 // non-null, non-released pointer when garbage collected. We use a tagged XPtr,
 // but do not set an XPtr finalizer
 Rcpp::XPtr<ArrowSchema> schema_owning_xptr(void) {
-  struct ArrowSchema* schema = (struct ArrowSchema*)ArrowMalloc(sizeof(struct ArrowSchema));
-  if (schema == NULL) Rcpp::stop("Failed to allocate ArrowSchema");
-  schema->release = NULL;
-  Rcpp::XPtr<ArrowSchema> schema_xptr = make_xptr(schema, false);
-  return schema_xptr;
+    struct ArrowSchema* schema = (struct ArrowSchema*)ArrowMalloc(
+        sizeof(struct ArrowSchema));
+    if (schema == NULL)
+        Rcpp::stop("Failed to allocate ArrowSchema");
```

Comment:
I would really really prefer if we stuck to existing coding convention which are to NOT use clang-format here.